Gruntfile.js

```diff
@@ -134,10 +134,10 @@ module.exports = function(grunt) {
 
   var browsers = [{
     browserName: "firefox",
-    version: "30"
+    platform: "linux"
   }, {
     browserName: "chrome",
-    version: "35"
+    platform: "linux"
```

Comment:
Why can't we switch to Windows and then just fix the tests in accordance in this PR.  I don't mark this currently as an upgrade